app/components/shared/HScroll.vue.ts

```diff
@@ -1,69 +0,0 @@
-import Vue from 'vue';
-import { Component } from 'vue-property-decorator';
-import ResizeSensor from 'css-element-queries/src/ResizeSensor';
-import isEqual from 'lodash/isEqual';
-
-export interface IHScrollModel {
-  canScroll: boolean;
-  canScrollLeft: boolean;
-  canScrollRight: boolean;
-}
-
-/**
- * horizontal scrolling wrapper
- */
-@Component({})
-export default class HScroll extends Vue {
```

Comment:
Never knew we had this component.  What was it used for?

Reply:
it was used when we had the top nav bar-- with the apps nav in that situation